### DISCUSSION TITLE
add option to disable anonymous system analytics as well

### DISCUSSION THREAD

**Discussion by lleo19** [hassio-addons/addon-unifi#143] [Ideas]
# Problem/Motivation

> The mission of Home Assistant is to limit data leaking as much as possible.

## Expected behavior

> This add-on asks if analytics should be disabled or enabled at the time of install. 

## Actual behavior

> When user chooses to disable analytics, no data should be sent to Ubiquiti, but from DNS logs and traffic analysis is evident that connections are made from the APs to "trace.svc.ui.com" address

## Steps to reproduce

> enable and monitor DNS requests from APs and add-on

## Proposed changes

> To completely disable analytics and anonymous reporting, another system setting is necessary in the unifi controller 
system.analytics.anonymous=disabled

More reference to this setting can be found here: [https://community.ui.com/questions/Pi-Hole-blocking-something-to-do-with-UI-com-trace-svc-ui-com/4a404478-63ef-4086-aa18-fbc431ce46f1](url)

**Comment by frenck** [comment]
Converted this issue into a discussion, as it isn't an issue; but an idea/feature request.

**Reply by lleo19** [reply]
thanks, so can this be added as an add-on configuration option?

**Reply by frenck** [reply]
I dunno, looking that the link provided, this depends on the site configuration. The add-on doesn't know anything about that and doesn't know which sites are configured. Thus is not able to create the right configuration files.

Otherwise, if Ubiquiti is doing data collection, they have to provide a decent option to disable that in the UI.
